benchmarks/experiment_runner.py

```diff
@@ -125,6 +125,10 @@ def generate_and_run_all_configs(self):
             logger.error("ERROR")
             self.save_results(dummy_benchmark_experiment, dummy_benchmark_model,
                               {"error": str(e)}, None)
+          except ValueError as e:
+            logger.error("ERROR")
```

Comment:
Can you log the error e, too?

Reply:
Yes, I used the `logger.exception` to do it automagically.